queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -210,6 +216,13 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
 
                 elif filing.get('correction'):
                     filing_submission = correction.process(filing_submission, filing, filing_meta)
+                    if business.state == Business.State.HISTORICAL:
```

Comment:
Which business scenario is this? If the business state is historical, can you file a correction?

Reply:
The business scenario is when a company is dissolved (by mistake), but staff needs to correct it back to ACTIVE.

Reply:
I think that action is called restoration. Correction is disabled for a historical entity unless this is a new requirement

Reply:
I've moved it to restoration.